From 087b7a1e16aaafcc8ce24715dc81cab9fa0999f4 Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Mon, 2 Mar 2015 15:48:03 +0800
Subject: [PATCH 0141/1221] MLK-10373-2 ARM: dtsi: imx7d: add dts support for
 the PWM

Add the required dts entries for the PWM
backlight module on the IMX7D board.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
Signed-off-by: Fancy Fang <chen.fang@freescale.com>
[zou:Original patch taken from
git.freescale.com/imx/fsl-arm-yocto-bsp.git-b imx-3.14.52-1.1.0_ga]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm/boot/dts/imx7d.dtsi |   16 ++++++++--------
 1 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index 55179d7..77f29b5 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -627,8 +627,8 @@
 				compatible = "fsl,imx7d-pwm", "fsl,imx27-pwm";
 				reg = <0x30660000 0x10000>;
 				interrupts = <GIC_SPI 81 IRQ_TYPE_LEVEL_HIGH>;
-				clocks = <&clks IMX7D_CLK_DUMMY>,
-					<&clks IMX7D_CLK_DUMMY>;
+				clocks = <&clks IMX7D_PWM1_ROOT_CLK>,
+					<&clks IMX7D_PWM1_ROOT_CLK>;
 				clock-names = "ipg", "per";
 				#pwm-cells = <2>;
 				status = "disabled";
@@ -638,8 +638,8 @@
 				compatible = "fsl,imx7d-pwm", "fsl,imx27-pwm";
 				reg = <0x30670000 0x10000>;
 				interrupts = <GIC_SPI 82 IRQ_TYPE_LEVEL_HIGH>;
-				clocks = <&clks IMX7D_CLK_DUMMY>,
-					<&clks IMX7D_CLK_DUMMY>;
+				clocks = <&clks IMX7D_PWM2_ROOT_CLK>,
+					<&clks IMX7D_PWM2_ROOT_CLK>;
 				clock-names = "ipg", "per";
 				#pwm-cells = <2>;
 				status = "disabled";
@@ -649,8 +649,8 @@
 				compatible = "fsl,imx7d-pwm", "fsl,imx27-pwm";
 				reg = <0x30680000 0x10000>;
 				interrupts = <GIC_SPI 83 IRQ_TYPE_LEVEL_HIGH>;
-				clocks = <&clks IMX7D_CLK_DUMMY>,
-					<&clks IMX7D_CLK_DUMMY>;
+				clocks = <&clks IMX7D_PWM3_ROOT_CLK>,
+					<&clks IMX7D_PWM3_ROOT_CLK>;
 				clock-names = "ipg", "per";
 				#pwm-cells = <2>;
 				status = "disabled";
@@ -660,8 +660,8 @@
 				compatible = "fsl,imx7d-pwm", "fsl,imx27-pwm";
 				reg = <0x30690000 0x10000>;
 				interrupts = <GIC_SPI 84 IRQ_TYPE_LEVEL_HIGH>;
-				clocks = <&clks IMX7D_CLK_DUMMY>,
-					<&clks IMX7D_CLK_DUMMY>;
+				clocks = <&clks IMX7D_PWM4_ROOT_CLK>,
+					<&clks IMX7D_PWM4_ROOT_CLK>;
 				clock-names = "ipg", "per";
 				#pwm-cells = <2>;
 				status = "disabled";
-- 
1.7.5.4

